build.gradle

```diff
@@ -106,7 +106,7 @@ test {
 task integrationTest(type: Test) {
     useJUnitPlatform()
     filter {
-        includeTestsMatching '*IntTest'
+        includeTestsMatching '*IntTest*'
```

Comment:
When running a single test in IntelliJ the test name would be of pattern TestClassNameIntTest.testName which doesn't match the above pattern. Probably there is another way to run tests individually. Will revert this for now.

Reply:
Ah ok, that's a fair point. That is a particular issue with the nested tests which I've just been working around but it would be better to fix it. Let's bring this back in in a future PR.